.genai/io/capstone/github_reviewer.py

```
@@ -49,35 +187,85 @@ def main():
+        if rag_context:
+            log_file.write(f"RAG CONTEXT:\n{rag_context}{separator}")
+        log_file.write(f"DIFFS:\n{file_diffs}{separator}")
+        log_file.write(f"RESPONSE:\n{response}{separator}")
 
+        # Process response
         if AiBot.is_no_issues_text(response):
             Log.print_green("File looks good. Continue", file)
         else:
             responses = AiBot.split_ai_response(response)
             if len(responses) == 0:
-                Log.print_red("Responses where not parsed:", responses)
+                Log.print_red("Responses were not parsed:", response[:200])
 
-            result = False
-            for response in responses:
-                if response.line:
-                    result = post_line_comment(github=github, file=file, text=response.text, line=response.line)
+            for comment in responses:
+                result = False
+                if comment.line:
+                    result = post_line_comment(
+                        github=github,
+                        file=file,
+                        text=comment.text,
+                        line=comment.line
+                    )
                 if not result:
-                    result = post_general_comment(github=github, file=file, text=response.text)
+                    result = post_general_comment(
+                        github=github,
+                        file=file,
+                        text=comment.text
+                    )
                 if not result:
-                    raise GitHubError("Failed to post any comments.")
+                    Log.print_red("Failed to post comment:", comment.text[:100])
 
 
-def post_line_comment(github: GitHub, file: str, text: str, line: int):
-    Log.print_green("Posting line", file, line, text)
+def post_line_comment(github: GitHub, file: str, text: str, line: int) -> bool:
+    """Post a comment to a specific line in the PR."""
```

267 : The script no longer raises an exception on a failed comment post, instead logging the error and continuing. While this makes the script more robust, it can lead to silent failures where review comments are not posted and the CI job still passes. Consider tracking the number of failed comments and exiting with a non-zero status code at the end of the script if any failures occurred. This would correctly signal a problem to the CI/CD system without halting execution on the first error.